pubpub/pubpub
# feature: Superadmin dash and `LandingPageFeature` manager

This PR adds some new stuff that came out of the landing page refresh effort. I'm pulling it out of the `landing-page-refresh` branch to get a clean diff for the spam-fighting work.

- The concept of "superadmin users" (that will be all of us!)
- A new superadmin dashboard for us to manage PubPub-wide things
- The first tab of said dashboard, which exists to manage the Pubs and Communities that we feature on the landing page.

![image](https://user-images.githubusercontent.com/2208769/205152922-dfad9f2f-70bc-48ee-bf60-da1df9396f6a.png)

_Test plan:_

- Run the unit tests for managing landing page features: `npm run test-dev server/landingPageFeature`
- Make your account a superadmin in the dev database and check that you can use it to administrate any community
- Check that as a superadmin (and only then) you can visit `/superadmin/landingPageFeatures`
- Use this page to add/remove/manipulate featured items
